lib/cdo/script_constants.rb

```diff
@@ -284,7 +285,9 @@ module ScriptConstants
     CSD3 = "csd3".freeze,
     CSD4 = "csd4".freeze,
     CSD5 = "csd5".freeze,
-    CSD6 = "csd6".freeze
+    CSD6 = "csd6".freeze,
+
+    TEST = 'ui-test-course'.freeze
```

Comment:
I think you need this for scripts that are like CSF where the the script is the "course"

Reply:
I am not sure I agree with this. it makes sense for CSF script family names to be in here, because we use those to redirect to the latest version of Course A, for example. The course you're adding is more like CSP or CSD, however entries above like 'csd3' are only here to redirect you to the latest version of a particular unit of CSD3, which we do not want because it is deprecated behavior which we are trying to remove. If you want to try and prove experimentally whether this is neede, you could leave this here, get the code working just how you want it, and then try removing this line.

Reply:
I added both a course like CSD/CSP and one like CSF since the versioning test has tests for both. You can see there are 2 courses create and 4 scripts. 1 script for each course and 2 more scripts to be CSF style courses.

Reply:
ok, I see what's going on here -- the string `'ui-test-course'` is the family name of the script, as well as the course. I had only noticed that it was the family name of the course, which is why I was confused.

I would recommend changing the family name here, and on the scripts, to match the script name minus the version year, so `'ui-test-script-as-course'`, since this is the pattern we follow e.g. for CSF courses/scripts. It took me a while to realize that "script as course" means something like Course A. One alternative would be to call these 'ui-test-versioned-script', but only if, having heard it, you prefer that name too.